data/sql/updates/pending_db_world/rev_1607503600897995800.sql

```diff
@@ -0,0 +1,257 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1607503600897995800');
+insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esES','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');
+insert into `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) values('783','esMX','Ah, bien. Otro voluntario. Estamos recibiendo muchos de ustedes estos días. Espero que sean suficientes.$B$BLas tierras humanas están siendo amenazadas desde afuera, y muchas de nuestras fuerzas han sido preparadas en el extranjero. Esto, a su vez, deja espacio para que los grupos corruptos y sin ley prosperen dentro de nuestras fronteras.$B$BEs una batalla de muchos frentes, más que nosotros $N. Prepárese para una larga campaña.','18019');
```

Comment:
- convert string with numbers into integers
- regroup the INSERT INTO
- add DELETE FROM before insert

Reply:
Thank you very much, I didn't know.
Next time I will do it :)

Reply:
it's better to do it in this PR, you can easily edit the PR even it is opened.

Reply:
I see your edits, convert the string like `'793'` into integer like `793` without quotes and put a newline between the VALUES for a better visualization pls
just this